ol/cli/src/mgmt/management.rs

```diff
@@ -238,9 +280,21 @@ impl Node {
                 "failed to run 'ol', is it installed?",
             )
         } else {
-            let project_root = self.app_conf.workspace.source_path.clone().unwrap();
+            let project_root = match self.app_conf.workspace.source_path.clone() {
+                Some(p) => p,
+                None => {
+                    println!("ERROR: can't start pilot in dev mode. It doesn't seem like you have workspace.source_path set in 0L.toml. Exiting.");
```

Comment:
Shouldn't it be just "ERROR: It doesn't seem like you have workspace.source_path set in 0L.toml. Exiting." ?
The exit is not due to dev mode.